### ISSUE TITLE
Tailscale add-on

### ISSUE THREAD

**Issue by uSlackr** [custom-components/remote_homeassistant#186]
I'm trying to configure remote HA using the tailscale add-on on both Instances.  I've tried using both Tailscale IP and the internal IP, but neither works with a non-specific error message.  Would love to see a working config

**Comment by ronschaeffer** [comment]
Same here. Happy to test any updates.

**Comment by cegit55** [comment]
Have been trying to set this up myself and no joy so far - I provide the Tailscale IP and Port plus the Token, but it just says "Failed to connect to server".  Has anyone managed to get it to work?

**Comment by andrewt81** [comment]
same for me

**Comment by vanrace** [comment]
Guys this component only is able to report status of a tailscale connection. You will need a full implementation like this one https://github.com/tsujamin/hass-addons to run the actual vpn

**Comment by comatose1978** [comment]
I am using Zero Tier and it is working well.

**Comment by GAguado** [comment]
Did you manage to get remote HA running with Tailscale?  even with Tsujamin addon, I can not, It might not be properly configured. Should I install just the addon and remove the official TailScale?

**Comment by vanrace** [comment]
Yes I did, not using it anymore, so i don’t have any configuration details for you unfortunately

**Comment by azzeloof** [comment]
Hope this isn't too late, but I was able to get it working by setting `userspace-networking: false` in the tailscale config on the main node. Hope that helps somebody!

**Comment by Elemong** [comment]
> Hope this isn't too late, but I was able to get it working by setting `userspace-networking: false` in the tailscale config on the main node. Hope that helps somebody!

@azzeloof Thanks that finally managed to fix my problems have been trying for hours!
